src/cache/surface_albedo.jl

```diff
@@ -115,51 +115,61 @@ Calculate the direct surface albedo using the regression function of Jin et al.
 function surface_albedo_direct(α_model::RegressionFunctionAlbedo{FT}) where {FT}
     α_dir =
         (λ, cosθ, u) -> begin
-            # relative refractive index of water and air (n = n_w/n_a)
-            n = α_model.n
-
-            # refractive index of water for visible light
-            n0 = α_model.n0
-
-            # mean wave slope distribution width
-            σ = α_model.wave_slope(u)
-
-            # Fresnel reflectance (assuming equal contribution of the p-polorized and s-polarized components, and using the perfect dielectric medium approximation)
-            sinθ(cosθ) = sqrt(1 - cosθ^2)
-            rf_p(n, cosθ) =
-                (
-                    (n^2 * cosθ - sqrt(n^2 - sinθ(cosθ)^2)) /
-                    (n^2 * cosθ + sqrt(n^2 - sinθ(cosθ)^2))
-                )^2
-            rf_s(n, cosθ) =
-                (
-                    (cosθ - sqrt(n^2 - sinθ(cosθ)^2)) /
-                    (cosθ + sqrt(n^2 - sinθ(cosθ)^2))
-                )^2
-            rf(n, cosθ) = (rf_p(n, cosθ) + rf_s(n, cosθ)) / 2
-
-            # regression coefficients
-            p = α_model.p
-
-            # the regression function (J11, eq. 4)
-            f(cosθ, σ) =
-                (
-                    p[1] +
-                    p[2] * cosθ +
-                    p[3] * cosθ^2 +
-                    p[4] * cosθ^3 +
-                    p[5] * σ +
-                    p[6] * σ * cosθ
-                ) * exp(
-                    p[7] +
-                    p[8] * cosθ +
-                    p[9] * cosθ^2 +
-                    p[10] * σ +
-                    p[11] * σ * cosθ,
+            if cosθ <= 0
+                return 0
```

Comment:
```suggestion
                return zero(FT)
```

Reply:
Thanks! Is zero(FT) the same as FT(0)?

Reply:
No, zero(FT) is slightly better, probably because it is inferred and compiled